caluma-demo/app/routes/documents.js

```diff
@@ -0,0 +1,31 @@
+import Route from "@ember/routing/route";
+import { queryManager } from "ember-apollo-client";
+import gql from "graphql-tag";
+
+export default class DocumentsRoute extends Route {
+  @queryManager apollo;
+
+  async model() {
+    const documents = await this.apollo.query(
+      {
+        query: gql`
+          query {
+            allDocuments {
+              edges {
+                node {
+                  id
+                  form {
+                    name
+                    description
+                  }
+                }
+              }
+            }
+          }
+        `
+      },
+      "allDocuments.edges"
+    );
+    return documents;
```

Comment:
You can just return the awaited query directly since you don't use the `documents` variable

Reply:
I have changed this.